projects/packages/my-jetpack/_inc/components/connection-status-card/index.tsx

```diff
@@ -167,17 +192,19 @@ const getUserConnectionLineData: getUserConnectionLineDataType = ( {
 	if ( userConnectionData.currentUser?.isMaster ) {
 		userConnectionText = userConnectionData.currentUser?.wpcomUser?.display_name
 			? sprintf(
-					/* translators: placeholder is user name */
-					__( 'Connected as %1$s (Owner).', 'jetpack-my-jetpack' ),
-					userConnectionData.currentUser?.wpcomUser?.display_name
+					/* translators: %1$s is user name, %2$s is the user email */
+					__( 'Connected as %1$s (Owner) ( %2$s ).', 'jetpack-my-jetpack' ),
```

Comment:
I noticed the email here has spaces around it whereas the connected owner does not. Was this intentional?

![Screenshot 2025-02-05 at 11 43 46 AM](https://github.com/user-attachments/assets/a6865332-a006-448a-b24f-d1d08fec40c3)

Reply:
Not intentional, I'll remove those spaces!